packages/generator/src/parser/common/edmx-to-vdm.ts

```diff
@@ -333,7 +346,7 @@ export function transformComplexTypes(
       originalName: c.Name,
       factoryName: formatter.typeNameToFactoryName(typeName, reservedNames),
       fieldType: complexTypeFieldType(typeName),
-      properties: c.Property.map(p => {
+      properties: c.Property.filter(filterUnknownEdmTypes).map(p => {
```

Comment:
I add this to the refactor generator ticket. We have twice roughly the same logic for building the VDMProperty and VDMComplexType.